services/web-app/components/resource-card/resource-card-group.module.scss

```diff
@@ -16,7 +16,8 @@
     margin-bottom: 0;
   }
 
-  :global([class*='cds--lg:col-span-4']:nth-child(3n)) {
+  // items: 3,5,7,9...
+  :global([class*='cds--lg:col-span-4']:nth-child(2n + 3)) {
```

Comment:
@alisonjoseph check me on this, I think we want to skip the third car and every 2 cards after that so that's why I made this change? 
<img width="1464" alt="image" src="https://user-images.githubusercontent.com/40550942/197878735-152eddf6-4551-49d7-a006-4d73ae28d558.png">

Reply:
Thanks! That seems to be what we want!